oidc-authenticator/src/main/java/org/xwiki/contrib/oidc/auth/internal/OIDCClientConfiguration.java

```diff
@@ -338,6 +338,11 @@ public Map<String, Set<String>> getProviderMapping()
      */
     private static final String PROP_LOGOUT_REDIRECT_URL = "oidc.afterLogoutURL";
 
+    /**
+     * @since 2.17.5
```

Comment:
It's a bit strange that it's private, since this is definitely public information and that's what was done for the other fields (the `private` for `PROP_LOGOUT_REDIRECT_URL` looks like a mistake, as suggested by the `@since` which is a bit useless for a private field).